setup.py

```diff
@@ -58,7 +58,6 @@ def read(*names, **kwargs):
         "compas_rhino",
         "compas_blender",
         "compas_ghpython",
-        "compas_plotters",
```

Comment:
Are plotters dead?

Reply:
will move them to separate package like `compas_view2`. in fact i would like to propose the same for `compas_blender`, `compas_rhino` and `compas_ghpython`, but for a different reason. it will be increasingly difficult in the future to provide manageable support for different Rhino versions. supporting both Rhino 6 and 8 for example results in not being able to use many of the useful features of Rhino 8. when in a separate package we could have different versions to offer different support...